src/coreclr/jit/codegenarmarch.cpp

```diff
@@ -2450,7 +2450,9 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
         assert(!call->IsTailCall());
```

Comment:
The comment above (Generate a direct call to a non-virtual user defined or helper method) is wrong - copy&paste bug. It should be either deleted or changed to talk about indirect call.